### ISSUE TITLE
core: New BDF assigned upon rebinding

### ISSUE THREAD

**Issue by arter97** [opencxl/opencxl#137]
```
./cxl-util start -c fm -c switch -c sld-group -c mld-group -c host -c host-group --config-file configs/1vcs_4sld.yaml
./cxl-util fm unbind 0 0
./cxl-util fm bind 0 0 1
```

This creates a new BDF of 09:00.0.

**Comment by arter97** [comment]
```
[PciBusDriver] BDF              : 06:00.0
[PciBusDriver] Vendor ID        : 0x1DC5
[PciBusDriver] Device ID        : 0x11DC
[PciBusDriver] Class Code       : 0x050210
[PciBusDriver] Is Bridge        : No
[PciBusDriver] Parent BDF       : 02:03.0
[PciBusDriver] BAR0 Base Address: 0xFE800000
[PciBusDriver] BAR0 Size        : 131072
[PciBusDriver] BAR1 Base Address: 0x0
[PciBusDriver] BAR1 Size        : 0
[PciBusDriver] BAR2 Base Address: 0x0
[PciBusDriver] BAR2 Size        : 0
[PciBusDriver] BAR3 Base Address: 0x0
[PciBusDriver] BAR3 Size        : 0
[PciBusDriver] BAR4 Base Address: 0x0
[PciBusDriver] BAR4 Size        : 0
[PciBusDriver] BAR5 Base Address: 0x0
[PciBusDriver] BAR5 Size        : 0
[PciBusDriver] Capabilities        :
[PciBusDriver]  - MSI Capability: 0x40
[PciBusDriver]  - PCI_EXPRESS Capability: 0x58
[PciBusDriver]     - Port Number: 0
[PciBusDriver]     - Device/Port Type: PCI_EXPRESS_ENDPOINT
[PciBusDriver]  - DEVICE_SERIAL_NUMBER Extended Capability: 0x100
[PciBusDriver]  - DESIGNATED_VENDOR_SPECIFIC Extended Capability: 0x10C
[PciBusDriver]  - DESIGNATED_VENDOR_SPECIFIC Extended Capability: 0x158
[PciBusDriver]  - DESIGNATED_VENDOR_SPECIFIC Extended Capability: 0x178
[PciBusDriver]  - DATA_OBJECT_EXCHANGE Extended Capability: 0x194
```

This one should be most likely removed.

**Comment by arter97** [comment]
VID and DID needs to be set to 0xFFFF to indicate it's an unused slot.

**Comment by kyeyoon-eeum** [comment]
With the latest change, the device BDF problem is fixed, but a new one surface (#142). Closing this ticket.
